diff --git a/scripts/qemucommand.py b/scripts/qemucommand.py
index 4abfd4e..cddaaf1 100644
--- a/scripts/qemucommand.py
+++ b/scripts/qemucommand.py
@@ -113,7 +113,7 @@ class QemuCommand(object):
         if self.gui:
             cmdline += ["-serial", "stdio"]
         else:
-            cmdline.append('-nographic')
+            cmdline += ['-nographic', "-serial", "mon:stdio"]
         if self.kvm:
             cmdline += ['-enable-kvm', '-cpu', 'host']
         else:
@@ -127,6 +127,8 @@ class QemuCommand(object):
             "qemu-img", "create",
             "-o", "backing_file=%s" % self.image,
             "-f", "qcow2",
-            self.overlay]
+            self.overlay,
+            "500M"
+            ]
         return cmdline
